src/main/java/com/example/UserModule/service/serviceImpl/UserServiceImplementation.java

```diff
@@ -147,12 +147,12 @@
     GeoIP geoIP = new GeoIP();
 
     if (!Objects.nonNull(user)) {
-      return new SignInResponseDto(ResponseMessages.USER_UNAVAILABLE, "Try Sign-UP !");
+      return new SignInResponseDto(ResponseMessages.USER_UNAVAILABLE, MessageStrings.TRY_SIGN_UP_TOKEN);
     }
```

Comment:
**Suggestion:** Consider checking if the user is null directly instead of using Objects.nonNull. It's more readable and straightforward.
```suggestion
    if (user == null) {
      return new SignInResponseDto(ResponseMessages.USER_UNAVAILABLE, MessageStrings.TRY_SIGN_UP_TOKEN);
    }
```